### ISSUE TITLE
Custom folders in /custom aren't ignored from git

### ISSUE THREAD

**Issue by filliph** [mattmc3/mac.bak#1]
There should perhaps not be any folders in `/custom` by default, so that subfolders of `/custom` can be added to `.gitignore`.

**Comment by mattmc3** [comment]
The point of `custom` is for folks to add additional backups that aren't handled by mackup, so I'm not sure what we'd accomplish by emptying it out. I'm okay with the community adding additional things in here and making a PR. But, maybe there's two concepts here - personal custom and global custom?

**Comment by filliph** [comment]
Yeah, that's where my confusion was; I thought the `custom` folder was for personal custom.

Want me to update the PR moving the new folders into the `custom` folder?
